README.md

```diff
@@ -87,6 +87,8 @@ We are providing example apps for Swift and Objective-C. These apps demonstrate
 
 Gini Vision Library can either be installed by using CocoaPods or by manually dragging the required files to your project.
 
+**Note on iOS 10:** No matter which option you choose if you want to support iOS 10 you need to specify the `NSPhotoLibraryUsageDescription` key in your `Info.plist` file. And if you're using the [Gini iOS SDK](https://github.com/gini/gini-sdk-ios) you need to add support for "Keychain Sharing" in your entitlements by adding a `keychain-access-groups` value to your entitlements file.
```

Comment:
Regarding `NSPhotoLibraryUsageDescription `... I remember there were two keys - one for the Camera and one for the Photo Library. If you don't specify them, you either fail review, or you get a crash when accessing that resource.  So, first of all, are you sure we don't actually need `NSCameraUsageDescription`? Secondly, maybe we should also add text explaining why this key is needed